schemas/Base64.json

```diff
@@ -3,5 +3,5 @@
   "title": "Base64",
   "description": "Base64 string",
   "type": "string",
-  "pattern": "^(?:[A-Za-z0-9+\/]{4}\\n?)*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)$"
+  "pattern": "^(?:[A-Za-z0-9+\/]{4}\\n?)*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$"
```

Comment:
Looking at more examples, it looks like some of the ECDSA curve points contain `_` in their base64 encoded form, which will fail validation according to our Base64.json schema, I may need to re-think this.